src/parser/xasl_generation.c

```diff
@@ -2822,6 +2830,100 @@ pt_split_attrs (PARSER_CONTEXT * parser, TABLE_INFO * table_info, PT_NODE * pred
   return ER_FAILED;
 }
 
+/*
+ * pt_split_hash_attrs () - Split the attr_list into two lists without destroying
+ *      the original list
+ *   return:
+ *   parser(in):
+ *   table_info(in):
+ *   pred(in):
+ *   build_attrs(out):
+ *   probe_attrs(out):
+ *
+ * Note :
+ */
+static int
+pt_split_hash_attrs (PARSER_CONTEXT * parser, TABLE_INFO * table_info, PT_NODE * pred, PT_NODE ** build_attrs,
+		     PT_NODE ** probe_attrs)
+{
+  PT_NODE *tmp = NULL, *pointer = NULL;
+  PT_NODE *build_nodes = NULL;
+  PT_NODE *probe_nodes = NULL;
+  int cur_build, num_attrs, i;
+  PT_NODE *attr_list = NULL;
+  PT_NODE *node = NULL, *save_node = NULL, *save_next = NULL;
+  PT_NODE *arg1, *arg2;
+  bool has_reserved = false;
+
+  build_nodes = NULL;		/* init */
+  *build_attrs = NULL;
+  *probe_attrs = NULL;
+
+  if (table_info->attribute_list == NULL)
+    return NO_ERROR;		/* nothing to do */
```

Comment:
even it is a single line, we need to enclose with a brace
```suggestion
  if (table_info->attribute_list == NULL)
    {
      return NO_ERROR;		/* nothing to do */
    }
```

Reply:
yes. I'll fix it